File: VonexDealer.Vue/ClientApp/models/NewPSTN.js
export class NewPSTN {
    constructor({
        newPSTNID = 0,
        contractPeriod = null,
        addressID = 0,
        termination = null,
        fee = 0.0,
        ratePlanID = 0
    } = {}) {
        this.newPSTNID = newPSTNID;
        this.addressID = addressID;
        this.termination = termination;
        this.fee = fee;
        this.ratePlanID = ratePlanID;
        this.contractPeriod = contractPeriod;
        this.ratePlanID = ratePlanID;
    }

}
export function createNewPSTN(data) {
    return Object.freeze(new NewPSTN(data));
}

File: VonexDealer.API/Models/Order/NewPSTN.cs
using System;

namespace VonexDealer.API.Models.Order
{
    public class NewPSTN
    {
        public Int64  NewPSTNID { get; set; }
        public Int64 AddressID { get; set; }
        public String Termination { get; set; }
        public double Fee { get; set; }
        public Int64 RatePlanID { get; set; }
        public Int64 LandlineID { get; set; }
        public int ContractPeriod { get; set; }
    }


}